docs/en/deployment/distributed-deployment-guide.md

```diff
@@ -1 +1,1409 @@
+
+### 3.1.8 emergencyPublish.supported.envs
+
+Configure a list of environments that allow emergency publishing, with multiple envs separated by commas.
+
+When the config service turns on the only one person can modify switch (`namespace.lock.switch`) for one publish, only one person can modify and another publish a configuration publish. In order to avoid not being able to publish the configuration in case of emergency (e.g. out-of-hours, holidays), you can configure this to allow some environments to operate emergency publishing, i.e. the same person can modify and publish the configuration.
+
+### 3.1.9 configView.memberOnly.envs
+
+A list of environments that display configuration information only to project members, with multiple envs separated by commas.
+
+For environments that are set to display configuration information only to project members, only the administrator of the project or users with edit or publish privileges for that namespace can see the configuration information and publish history for that private namespace. Public namespaces are always visible to all users.
+
+> Supported since version 1.1.0, see [PR 1531](https://github.com/apolloconfig/apollo/pull/1531)
+
+### 3.1.10 role.create-application.enabled - whether to enable control over creating project permissions
+
+> For versions 1.5.0 and above
+
+Default is false, all users can create projects
+
+If set to true, only super administrators and accounts with create-application privileges can create projects. Super administrators can assign project creation privileges to users via `Administrator Tools - System Rights Management`.
+
+### 3.1.11 role.manage-app-master.enabled - Enables or disables the project administrator to assign privileges
+
+> For versions 1.5.0 and above
+
+Default is false, all project administrators can add/remove administrators for the project
+
+If set to true, only super administrators and accounts with project administrator assignment privileges can add/remove administrators for a specific project, and super administrators can assign project-specific administrator assignment privileges to users via `Administrator Tools - System Rights Management
+
+### 3.1.12 admin-services.access.tokens - set the access token required by apollo-portal to access the apollo-adminservice for each environment
+
+### 3.1.13 searchByItem.switch - whether the console search box supports searching by configuration item
+
+The default is true, which makes it easy to quickly search for configurations by configuration item
+
+If set to false, this feature is disabled 
+
+> for version 1.7.1 and above
+
+If the corresponding environment apollo-adminservice has [access control enabled](#_326-admin-services.access.control.enabled-Configure-whether-apollo-adminservice-has-access-control-enabled), then you need to configure apollo-portal access here access token required for this environment apollo-adminservice, otherwise access will fail
```

Comment:
```suggestion
If the corresponding environment apollo-adminservice has [access control enabled](#_326-admin-servicesaccesscontrolenabled-configure-whether-apollo-adminservice-has-access-control-enabled), then you need to configure apollo-portal access here access token required for this environment apollo-adminservice, otherwise access will fail
```